package.json

```diff
@@ -66,7 +66,7 @@
     "start-docker": "node --max_old_space_size=4096 ./node_modules/webpack-dev-server/bin/webpack-dev-server.js  --config webpack.docker.config.js",
     "start-prod": "cross-env NODE_ENV=production NETWORK=main node --max_old_space_size=4096 ./node_modules/webpack-dev-server/bin/webpack-dev-server.js  --config webpack.dev.config.js",
     "start-staging": "cross-env NODE_ENV=staging NETWORK=rinkeby node --max_old_space_size=4096 ./node_modules/webpack-dev-server/bin/webpack-dev-server.js --config webpack.dev.config.js",
-    "start-xdai": "cross-env NODE_ENV=production NETWORK=xdai node --max_old_space_size=4096 ./node_modules/webpack-dev-server/bin/webpack-dev-server.js --config webpack.prod.config.js",
+    "start-xdai": "cross-env NODE_ENV=staging NETWORK=xdai node --max_old_space_size=4096 ./node_modules/webpack-dev-server/bin/webpack-dev-server.js --config webpack.dev.config.js",
```

Comment:
?

Reply:
when we run xdai locally we do it like we do when we run staging locally.

Reply:
yeah - so these commands are not used for deployment, they are just conveient for running locally...